django_scrubber/migrations/0001_initial.py

```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
```suggestion
```
I would remove all of those useless `# -*- coding: utf-8 -*-`

Reply:
Agreed for "custom" files. I personally wouldn't change ones generated by Django like migrations.

Reply:
Not the case anymore if you use Django 3

Reply:
Yup. But the project wasn't started with Django 3...

Reply:
removed in the non-generated files; left this one as is :heavy_check_mark: